scripts/deploy_assisted_service.sh

```diff
@@ -146,6 +147,15 @@ EOF
 else
     print_log "Updating assisted_service params"
 
+    if [[ "${PLATFORM}" == "none"  || "${PLATFORM}" == "external" ]]; then
+        # on RHEl9 we need to open ports in a new policy
+        # between libvirt and HOST
+        print_log "Opening additional ports for none/external"
+        firewall-cmd --policy=libvirt-to-host --add-port={22623/tcp,6443/tcp}
```

Comment:
```suggestion
        firewall-cmd --policy=libvirt-to-host --add-port={22623/tcp,22624/tcp,6443/tcp}
```
could be worth t add the MCS/http port as well

Reply:
will it also work on rhel8 ? maybe add a condition to act only in case of rocky9/rhel9 ?